packages/plugin-ext/src/hosted/node/plugin-reader.ts

```diff
@@ -36,15 +36,20 @@ export class HostedPluginReader implements BackendApplicationContribution {
     private readonly scanner: MetadataScanner;
 
     @optional()
-    @multiInject(MetadataProcessor) private readonly metadataProcessors: MetadataProcessor[];
+    @multiInject(MetadataProcessor)
+    private readonly metadataProcessors: MetadataProcessor[];
+
+    @optional()
+    @multiInject(PluginResourcesProvider)
```

Comment:
never use `multiInject`, use `ContributionProvider`, that's the documented way to introduce an extension point and the real type is `PluginResourcesProvider[] | undefined` then there is no a single binding, see https://github.com/theia-ide/theia/pull/4774#discussion_r270945494 for more

I will update the coding guidelines.

Reply:
added to the coding guidelines: https://github.com/theia-ide/theia/wiki/Coding-Guidelines#no-multi-inject